src/cmd/group.go

```diff
@@ -38,9 +39,161 @@ EOF
 	},
 }
 
-// TODO: Get Group
+var getGroupCommand = &cobra.Command{
+	Use:        "group ID|ALIAS",
+	Short:      "Get details about a group",
+	Long:       `Get details about a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		common.PrettyPrint(group)
+		common.WasFound(group.Id == nil, key)
+	},
+}
 
-// TODO: List Groups
+var getMembersCommand = &cobra.Command{
+	Use:        "members ID|ALIAS",
+	Short:      "Get members for a group",
+	Long:       `Get members for a group`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		var group *opslevel.Group
+		var err error
+		if common.IsID(key) {
+			group, err = getClientGQL().GetGroup(key)
+		} else {
+			group, err = getClientGQL().GetGroupWithAlias(key)
+		}
+		cobra.CheckErr(err)
+		members, err := group.Members(getClientGQL())
+		cobra.CheckErr(err)
+		common.PrettyPrint(members)
+		common.WasFound(group.Id == nil, key)
+	},
+}
+
+var getDescendantRepositoriesCommand = &cobra.Command{
+	Use:        "repositories ID|ALIAS",
+	Short:      "Get descendant repositories for a group",
```

Comment:
We should probably use the descriptions in the API spec

In this case specifically
```
All the repositories that fall under this group - ex. this group's child repositories, all the child repositories of this group's descendants, etc.
```
Maybe just for the Long description and then for the short lets do something like
```
Get all repositories for a group
```